src/dataexplorer/dataexplorer-application/src/components/Navigation/Navigation.tsx

```diff
@@ -1,83 +1,104 @@
-import React from 'react';
-import { Menu, Layout } from 'antd';
+import React, { useState } from 'react';
+import { Layout, Avatar } from 'antd';
+import Breadcrumbs from './Breadcrumbs/Breadcrumbs';
 import 'antd/dist/antd.css';
-import Dashboard from '../Dashboard/Dashboard';
-import MySchemas from '../MySchemas/MySchemas';
-import MyPatients from '../MyPatients/MyPatients';
-import { Switch, Route, Link, withRouter } from 'react-router-dom';
-import MenuItem from 'antd/lib/menu/MenuItem';
+import MyPatients from '../PatientInfo/MyPatients';
+import { Switch, Route, withRouter, Link } from 'react-router-dom';
+import { UserOutlined } from '@ant-design/icons';
+import PatientInfo from '../PatientInfo/PatientInfo';
 
-const { Header, Content, Sider, Footer } = Layout;
-const data = {
-    items: [
-        {
-            id: 1,
-            url: '/',
-            title: 'Hjem',
-        },
-        {
-            id: 2,
-            url: '/Skjema',
-            title: 'Mine Skjema',
-        },
-        {
-            id: 3,
-            url: '/Pasient',
-            title: 'Mine pasienter',
-        },
-        {
-            id: 4,
-            url: '/Search',
-            title: 'Filtrer på egenskaper',
-        },
-    ],
-};
+const { Header, Content } = Layout;
+const Navigation = (props: any) => {
+    const [name, setName] = useState('');
+    const [schemaNumber, setSchemaNumber] = useState('');
 
-const Navigation = () => (
-    <>
-        <Layout>
-            <Sider breakpoint="xs" collapsedWidth="0">
-                <div className="logo">
-                    <h1
+    return (
+        <>
+            <Layout>
+                <Layout>
```

Comment:
Any reason this is wrapped in two layouts, and not only one?

Reply:
No, nice catch.